Content.Server/Recycling/RecyclerSystem.cs

```diff
@@ -1,49 +1,91 @@
+using Content.Server.Audio;
 using Content.Server.Power.Components;
 using Content.Server.Recycling.Components;
+using Content.Shared.Audio;
 using Content.Shared.Body.Components;
 using Content.Shared.Recycling;
-using Robust.Shared.GameObjects;
-using Robust.Shared.IoC;
+using Content.Shared.Tag;
+using Robust.Shared.Audio;
 using Robust.Shared.Physics.Dynamics;
+using Robust.Shared.Player;
+using Robust.Shared.Timing;
 
 namespace Content.Server.Recycling
 {
-    internal sealed class RecyclerSystem : EntitySystem
+    public sealed class RecyclerSystem : EntitySystem
     {
+        [Dependency] private readonly IGameTiming _timing = default!;
+        [Dependency] private readonly AmbientSoundSystem _ambience = default!;
+        [Dependency] private readonly TagSystem _tags = default!;
+
+        private const float RecyclerSoundCooldown = 0.8f;
+
         public override void Initialize()
         {
-            base.Initialize();
-            SubscribeLocalEvent<RecyclerComponent, StartCollideEvent>(HandleCollide);
+            SubscribeLocalEvent<RecyclerComponent, StartCollideEvent>(OnCollide);
+        }
+
+        public void EnableRecycler(RecyclerComponent component)
+        {
+            if (component.Enabled) return;
+
+            component.Enabled = true;
+            _ambience.SetAmbience(component.Owner, true);
+        }
+
+        public void DisableRecycler(RecyclerComponent component)
+        {
+            if (!component.Enabled) return;
+
+            component.Enabled = false;
+            _ambience.SetAmbience(component.Owner, false);
         }
 
-        private void HandleCollide(EntityUid uid, RecyclerComponent component, StartCollideEvent args)
+        private void OnCollide(EntityUid uid, RecyclerComponent component, StartCollideEvent args)
         {
+            if (component.Enabled && args.OurFixture.ID != "brrt") return;
+
             Recycle(component, args.OtherFixture.Body.Owner);
         }
 
         private void Recycle(RecyclerComponent component, EntityUid entity)
         {
-            // TODO: Prevent collision with recycled items
+            RecyclableComponent? recyclable = null;
 
             // Can only recycle things that are recyclable... And also check the safety of the thing to recycle.
-            if (!EntityManager.TryGetComponent(entity, out RecyclableComponent? recyclable) || !recyclable.Safe && component.Safe) return;
+            if (!_tags.HasTag(entity, "Recyclable") &&
+                (!TryComp(entity, out recyclable) || !recyclable.Safe && component.Safe))
+            {
+                return;
+            }
+
+            // TODO: Prevent collision with recycled items
 
             // Mobs are a special case!
             if (CanGib(component, entity))
             {
```

Comment:
I out of scoped proper gibbing because ideally it just applies a lot of damage and doesn't instakill players.